src/Build/Evaluation/Evaluator.cs

```diff
@@ -1785,8 +1785,7 @@ private List<ProjectRootElement> ExpandAndLoadImports(string directoryOfImportin
 
                 if (!sdkResult.Success)
                 {
-                    // Ignore the missing import if IgnoreMissingImports is set unless FailOnUnresolvedSdk is also set
-                    if (_loadSettings.HasFlag(ProjectLoadSettings.IgnoreMissingImports) && !_loadSettings.HasFlag(ProjectLoadSettings.FailOnUnresolvedSdk))
+                    if (_loadSettings.HasFlag(ProjectLoadSettings.IgnoreMissingImports) && (!ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_0) || !_loadSettings.HasFlag(ProjectLoadSettings.FailOnUnresolvedSdk)))
```

Comment:
I don't think I understand this. Why is it not wave 16.10, the one for the release this is targeting? What is this expression representing? Why not do this the way it was done in the prior commit and condition adding the new flag on the changewave?

Reply:
Will change to 16.10. I can also retarget this to vs16.10, since it's currently at master.

I think making the change in either place should be equivalent. I thought this would be cleaner. If you don't like it, I can change to the other way, though.